### PR TITLE
Option Page styled #9

### PR DESCRIPTION
**Pull request by fabrizvi**
I styled the Options page according to me (I am not good at designing).
It looks decent now and is responsive even :)
PS: I don't know how to make PR for existing issue thats why made a new one with PR - I am new to the PR stuff, hope you don't mind.

Here are some screenshots
scroll1
scrroll2

### CODE REVIEW

**Comment by prateek3255** [comment]
This seems nice, but can you please replace the yellow color with white.

**Comment by prateek3255** [comment]
Also, we're planning to show the title of the page and the percentage complete once #11 gets merged. So I'll hold this till then.

**Comment by fabrizvi** [comment]
Okay I will change Yellow to white till then.

**Comment by fabrizvi** [comment]
It looks like this now,
lemme know if u need some changes. 
Cheers 😊
scroll1
scrroll2

**Comment by prateek3255** [comment]
I have updated the extension with the percentage complete changes. So please update your branch accordingly so that we can merge it.

**Comment by fabrizvi** [comment]
Hey Before committing I want to ask If you like this design better then Black and white. Because you changed your Icons to color Blue so Black doesn't look that good. I used some Adobe color palette and made this 👇
scroll1
scrroll2
Do lemme know If you are Okay with these or just want Black and white.

**Comment by prateek3255** [comment]
This looks nice ✨ Can you please also remove the icon from header, I'll run this locally and see how this works.

**Comment by fabrizvi** [comment]
On it!

File: options.css [hunk 1]
```diff
@@ -1,21 +1,110 @@
...
+	color: #fff;
+	font-weight: bolder;
+	/* min-width: 250px; */
+}
+
+.bookmark p {
+	overflow: hidden;
+	white-space: nowrap;
+	text-overflow: ellipsis;
+	font-family: cursive;
```
Comment (prateek3255) [inline options.css]: The cursive font looks a bit weird can you please remove it.
image

**Comment by prateek3255** [comment]
Also, I think you have opened PR from the wrong branch, the branch where your latest changes are is [Improvement-in-Options-Page](https://github.com/fabrizvi/scrroll-in/tree/Improvement-in-Options-Page), but you have opened a PR from [Improvement-fot-options-page](https://github.com/fabrizvi/scrroll-in/tree/Improvement-fot-options-page) you'll need to close this PR and open a new one from the former branch.

**Comment by prateek3255** [comment]
Also, can you try playing around with colors to match the theme with #35

**Comment by prateek3255** [comment]
@fabrizvi Please let me know if you're stuck anywhere, I have merged a PR for #35, and I also want to merge yours before releasing the next version.

**Comment by fabrizvi** [comment]
@prateek3255 I am on it gimme 10 minutes, please.

**Comment by prateek3255** [comment]
I am not in a hurry. I was asking if you are facing some issues you can feel free to ask me. You can finish it on your own pace 🤗

**Pull request closed without merging**